**Problems I faced:**
1. SignatureDoesNotMatch in generate_presigned_url.py while using 